draft-irtf-cfrg-hpke.md

```diff
@@ -786,10 +846,10 @@ any IND-CCA2 scheme, and the DH group and KDF satisfy the following
 conditions:
 
 - DH group: The gap Diffie-Hellman (GDH) problem is hard {{GAP}}.
-- Hash: Collision resistance.
-- Extract: Indifferentiable from a random oracle.
-- Expand: Behaves as a pseudorandom function wherein the first argument
-is the key.
+- Extract and Extract_kem: Indifferentiable from a random oracle,
+  respectively.
```

Comment:
What does "respectively" mean here?  If you mean that they are each individually indifferentiable, I would phrase as "Each function is individually indifferentiable from a random oracle".

Reply:
I assume this bullet and the next one are supposed to be "KDF used in DHKEM" and "KDF used in the rest of HPKE"?  If so, let's just say that.  Something like:

```
- Extract and Expand (in DHKEM): ...
- Extract and Expand (elsewhere): ...
```

Reply:
Ups, sorry that one slipped through after removing the `_kem` suffix. Yes, it means “Each function […]”.

Reply:
Addressed in commit https://github.com/cfrg/draft-irtf-cfrg-hpke/pull/50/commits/fbea8c041a50b6db3a458b7c2cdefa4cdb1014e8.